common/Android.mk

```diff
@@ -88,6 +87,11 @@ LOCAL_SRC_FILES := \
         utils/hwcutils.cpp \
         utils/disjoint_layers.cpp
 
+ifeq ($(strip $(ENABLE_NESTED_DISPLAY_SUPPORT)), true)
```

Comment:
Just leave things as it is. I dont think we want to do this change.

Reply:
no these changes, xen based android can't work, it's the reason  revert mingwei's nested display patch

Reply:
please check my comments in second patch. We want to only have IOCTL related stuff only in this macro nothing else.